configs/LLNL-Tioga-HPECray-zen3-MI250X-Slingshot/auxiliary_software_files/packages.yaml

```diff
@@ -193,6 +193,13 @@ packages:
       prefix: /opt/cray/pe/libsci/23.05.1.4/gnu/10.3/x86_64/
   lapack:
     buildable: false
+  hypre:
```

Comment:
Please remove hyper from this list.  All variants are already set to amdgpu_target=gfx90a